packages/core/src/html/scriptAndStyleTagProcessor.ts

```diff
@@ -10,24 +11,23 @@ const cheerio = require('cheerio');
  * @param node {Object<any, any>} node from the dom traversal
```

Comment:
Is `Object<any, any>` an accurate description of `DomElement` ?

Shall we remove the type here in JSdocs as that information is now available directly in TypeScript.

Reply:
Right, thanks for the catch!